pkgs/applications/science/math/mathematica/default.nix

```diff
@@ -34,139 +42,101 @@
 , xkeyboard_config
 , xorg
 , zlib
+# settings and optional packages
+, cudaSupport ? false
+, cudaPackages ? null
 , lang ? "en"
+, version ? null
```

Comment:
```suggestion
, version ? ""
```

Reply:
`null` and `""` for version is for the same purpose, this is left for open discussion though.

Reply:
> `null` and `""` for version is for the same purpose, this is left for open discussion though.

I think `null` is a harsher version of `""` which might make it easier for us to catch errors in the future.